static/app/views/dashboards/manage/index.tsx

```diff
@@ -57,6 +60,16 @@ function ManageDashboards() {
   const navigate = useNavigate();
   const location = useLocation();
   const api = useApi();
+  const dashboardGridRef = useRef<HTMLDivElement>(null);
+
+  const [showTemplates, setShowTemplatesLocal] = useLocalStorageState(
+    SHOW_TEMPLATES_KEY,
+    shouldShowTemplates()
+  );
+  const [_, setGridWidth] = useState(0);
```

Comment:
🤔 since this value is not used, what is the purpose of `setGridWidth`?

Reply:
setGridWidth is used to set the resizing logic here 
```
      setGridWidth(prevWidth => {
            if (currentWidth !== prevWidth && prevWidth !== 0) {
              setResizing(true);
              return currentWidth;
            }
            return prevWidth;
          });
```

Reply:
Right but if the value isn't used, why bother setting this state?